### DISCUSSION TITLE
paddingX, paddingY props

### DISCUSSION THREAD

**Discussion by hharnisc** [twilio-labs/paste#595] [Ideas] [answered]
With the `Box` I tried to use `paddingX` and `paddingY` props from styled system: https://styled-system.com/api/ 

This caused an issue with Typescript, however the component rendered as expected. I can definitely work around by setting multiple padding props, but this seems like a nice to have!

**Comment by serifluous** [comment]
Thanks, @hharnisc! We're looking into this and will get back to you within a day.

**Comment by TheSisb** [comment]
I'm not opposed to adding it. We didn't add it initially because we thought it might add complexity and possible issues if folks do `paddingX` followed by a `paddingRight` on the same element. I don't think that's a huge concern anymore.

**Reply by serifluous** [reply]
Adding to our backlog 👍

**Reply by hharnisc** [reply]
Thank you 🙌

**Comment by TheSisb** [comment, marked as answer]
@hharnisc This is now live in newer versions of Box 🎉  (paddingX, paddingY, marginX, marginY)
Box now also takes typography props directly, this includes all the props that previously could only be passed into Text.
